src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs

```diff
@@ -127,15 +127,15 @@ public string GetCurrentRequestHeader(ApiEvent eventInProgress)
             var regionalFields = new StringBuilder();
             if (!string.IsNullOrEmpty(regionDiscovered))
```

Comment:
It doesn't matter if regionDiscover is true or false. We should not have any `if` branches here. If a field is empty, we should still log it. Otherwise there is no way on the DB side of things to make queries. 

So we should log in our private fields: 

regionDiscovered, regionSource, isTokenCacheSerialized

Then when we make queries, we always know that at position 1 there is regionDiscovered, at position 2 there is regionSource and so on.

Reply:
Same comment for `GetLastRequestHeader`